lte/gateway/c/core/oai/tasks/nas/emm/ServiceRequestHdl.c

```diff
@@ -70,10 +70,8 @@ static int check_paging_received_without_lai(mme_ue_s1ap_id_t ue_id);
 /****************************************************************************/
 status_code_e emm_proc_service_reject(const mme_ue_s1ap_id_t ue_id,
                                       const uint8_t emm_cause) {
-  status_code_e rc = RETURNok;
   OAILOG_FUNC_IN(LOG_NAS_EMM);
```

Comment:
The `emm_proc_service_reject` function is just a wrapper around `emm_service_reject`. Better to rename `emm_service_reject` to `emm_proc_service_reject` and get rid of this wrapper.

Reply:
I will raise as a part of separate PR.?

Reply:
Tracked in https://github.com/magma/magma/issues/13134